checker-qual/src/main/java/org/checkerframework/checker/mustcall/qual/CreatesMustCallFor.java

```diff
@@ -16,17 +16,17 @@
  * argument/element. More precisely, the method resets the target's must-call type to the least
  * upper bound of its current must-call type and its declared must-call type.
  *
- * <p>When calling a method annotated as {@code @CreatesObligation("}<em>target</em>{@code ")}, the
+ * <p>When calling a method annotated as {@code @CreatesMustCallFor("}<em>target</em>{@code ")}, the
```

Comment:
@kelloggm @Nargeshdb sorry for just thinking of this now, but did we want to do the annotation renaming as a separate PR that lands before this one?  It may make this one slightly easier to review, but not sure if it's worth it

Reply:
I agree that it would be slightly nicer to do the annotation renaming separately.

Reply:
It would be. I'll do that; this PR won't change, but I'll open another one that just renames the annotation in the existing code.